src/Table/index.stories.tsx

```diff
@@ -41,22 +41,31 @@ const data: Array<Person> = [
   },
 ];
 
-const FilterDropdown: ColumnProps<Person>['filterDropdown'] = () => (
-  <div>test</div>
-);
-
 const columns: Array<ColumnProps<Person>> = [
   {
     title: 'Name',
     dataIndex: 'name',
-    filterDropdown: FilterDropdown,
   },
   {
     title: 'Age',
     dataIndex: 'age',
   },
 ];
 
-export const Default: Story<TableProps<Person>> = (args) => (
-  <Table dataSource={data} columns={columns} rowKey="id" {...args} />
-);
+const Template: Story<TableProps<any>> = (args) => <Table {...args} />;
```

Comment:
```suggestion
const Template: Story<TableProps<Person>> = (args) => <Table {...args} />;
```

Reply:
Not sure how to solve this type issue then? Can you help me with this?
![image](https://user-images.githubusercontent.com/26680884/127634337-50bb88af-a2b8-496e-be9c-b1b92180c5a7.png)

Reply:
Please don't ignore issues or mask them by making the components usage less typesafe. We would have hit this problem once implementing the component in our code. Using proper types in tests validates the component types are well-defined.

Reply:
Ok, to me it looked like a storybook artefact due to the extraction of the template and not like a problem that would happen in the app code.. :-(